paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetResultCallback.kt

```diff
@@ -10,5 +10,5 @@ import com.stripe.android.ExperimentalCustomerSheetApi
 @ExperimentalCustomerSheetApi
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 fun interface CustomerSheetResultCallback {
-    fun onResult(result: CustomerSheetResult)
+    fun onResult(result: CustomerSheetResult?)
```

Comment:
Not the nullable result again!

When would this be null?

Reply:
Null when parcelizing :D

Reply:
Can we return an error result instead of null? Or even cancelled?